websites/S/South Park/dist/metadata.json

```diff
@@ -4,14 +4,18 @@
     "name": "Slowlife",
     "id": "374905512661221377"
   },
-  "url": "www.southparkstudios.com",
+  "url": [
+    "www.southparkstudios.com",
+    "www.southpark.lat",
+    "southpark.cc.com"
+  ],
   "description": {
     "en": "South Park. Every episode of this critically acclaimed and award-winning show is packed with unforgettable adventures, cutting satire, and hilarious absurdity",
     "nl": "South Park. Elke aflevering van deze veelgeprezen en bekroonde show zit boordevol onvergetelijke avonturen, scherpe satire en hilarische absurditeit",
     "es": "South Park. Cada episodio de este programa galardonado y aclamado por las críticas está repleto de aventuras inolvidables, una sátira grosera y absurdidad graciosa."
   },
   "service": "South Park",
-  "version": "1.0.7",
+  "version": "1.0.6",
```

Comment:
_bump lmao_
```suggestion
  "version": "1.0.8",
```

Reply:
uhm im not sure thats an upgrade but more like an downgrade because 1.0.6 is the same as 1.0.8 then